clang/test/CodeGenSYCL/infer-address-spaces.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN:  %clang_cc1 -O1 -fsycl-is-device -internal-isystem %S/Inputs -triple spir64 -emit-llvm %s -o - | FileCheck %s
+
```

Comment:
Please add a comment here about what the test does. Thank you

Reply:
Done.